cypress/e2e/e2e_tegb.cy.js

```diff
@@ -0,0 +1,65 @@
+import { faker } from "@faker-js/faker";
+import { LoginPage } from "../page-objects/login_page";
+import { LoginApi } from "../api/login_api";
+import { HomePage } from "../page-objects/home_page";
+import { AccountApi } from "../api/account_api";
+
+describe("Register, Login new User and create Account E2E test Tegb", () => {
+  let username, password, email, firstname, lastname, phonenumber;
+
+  beforeEach(() => {
+    username = faker.internet.userName();
+    password = faker.internet.password({ length: 15 });
+    email = faker.internet.exampleEmail();
+    firstname = faker.person.firstName();
+    lastname = faker.person.lastName();
+    phonenumber = faker.phone.number();
+    new LoginPage().openTegb();
+  });
+  it("Create new user with a new account test", () => {
+    const loginApi = new LoginApi();
+    const accountApi = new AccountApi();
+    cy.intercept("/tegb/profile").as("profile_api");
+    cy.intercept("/tegb/accounts").as("accounts_api");
+    new LoginPage()
+      .openTegb()
+      .clickregisterNewUser()
+      .typeNewUsername(username)
+      .typeNewPassword(password)
+      .typeNewEmail(email)
+      .clickRegister();
+    new LoginPage().openTegb();
+    loginApi.login(username, password).then((response) => {
```

Comment:
Tady to vnoření by šlo rozdělit (stačí vytvořit account a pak `then` ukončit. Opět to trochu pomůže čitelnosti